cli/repair/clusterwide_config.go

```diff
@@ -0,0 +1,302 @@
+package repair
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tarantool/cartridge-cli/cli/common"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	configDirName        = "config"
```

Comment:
This rattling mix of aligns can kill any perfectionist

Reply:
It's go formatter.